crates/router/src/core/payouts.rs

```diff
@@ -467,15 +466,17 @@ pub async fn payouts_update_core(
     )
     .await?;
 
-    payouts_core(
-        &state,
-        &merchant_account,
-        &key_store,
-        &mut payout_data,
-        req.routing.clone(),
-        req.connector.clone(),
-    )
-    .await?;
+    if let Some(true) = payout_data.payouts.confirm {
```

Comment:
why do we have to have this validation ? since it's confirm flow it is already inferred that confirm is true.

Reply:
+1

Reply:
This is update flow - https://github.com/juspay/hyperswitch/pull/5248/files/c723092d1c9f1e2888f9b6648dacfeafe559216c#diff-8a9ac8080278922e1a240e07553e19178a2f05639d03cb3e914e28ded1252954R418